lib/spectacles.rb

```diff
@@ -40,4 +40,13 @@ def trailer(stream)
   end
 end
 
+ActiveRecord::ConnectionAdapters::SchemaStatements.class_eval do
+  alias_method(:_spectacle_drop_table, :drop_table) if method_defined?(:drop_table)
+
+  # Add this so that views get cleaned up properly
+  def drop_table(name, options={})
+    execute "DROP TABLE #{name} CASCADE"
```

Comment:
I considered that, however, the base drop_table code in AR explicitly ignores options:

``` 
rails/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb:361

      # Drops a table from the database.
      #
      # Although this command ignores +options+ and the block if one is given, it can be helpful
      # to provide these in a migration's +change+ method so it can be reverted.
      # In that case, +options+ and the block will be used by create_table.
      def drop_table(table_name, options = {})
        execute "DROP TABLE #{quote_table_name(table_name)}"
      end
```